src/Donations/Models/Donation.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace Give\Donations\Models;
+
+use DateTime;
+use Give\Donors\Models\Donor;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Subscriptions\Models\Subscription;
+
+/**
+ * Class Donation
+ *
+ * @unreleased
+ */
+class Donation
+{
+    /**
+     * @var int
+     */
+    public $id;
+    /**
+     * @var int
+     */
+    public $sequentialId;
+    /**
+     * @var DateTime
+     */
+    public $createdAt;
+    /**
+     * @var DateTime
+     */
+    public $updatedAt;
+    /**
+     * @var string
+     */
+    public $status;
+    /**
+     * @var int
+     */
+    public $amount;
```

Comment:
@alaca Is it the total amount?

Reply:
Yes, @ravinderk. This is the total amount.